content/en/synthetics/private_locations/configuration.md

```diff
@@ -97,11 +97,20 @@ Discard SSL errors when private location is using a proxy to send requests to Da
 **Default**: `10`<br>
 Maximum number of tests executed in parallel.
 
+`enableStatusProbes`
+: **Type**: <br>
+Enables the readiness and liveness of private location probes. 
```

Comment:
```suggestion
Enables the readiness and liveness of private location probes. This will enable two endpoints, http://localhost:8080/liveness and http://localhost:8080/readiness
```

I'm not sure if we prefer to say `localhost` or `127.0.0.1`, linking in @m-rousse for a second (experienced) opinion.

Reply:
I'd vote for `127.0.0.1`, though both are OK